### PR TITLE
added a migration note to the release notes IRT tuple instances

### PR DESCRIPTION
**Pull request by kailuowang**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -6,10 +6,12 @@ Version 0.5.0 is the fifth release.
 
 This release includes some API changes:
 
+`cats.laws.discipline.eq` no longer provides `Eq` instances for `tuple2` and `tuple3`, these instances and together with some other new instances for `tuple`s are not provided by `cats.std.tuple` (through inheriting the instance trait defined in algebra 0.4.2).
```
Comment (non) [inline CHANGES.md]: This is a typo right? Should be "are now provided by" right?

File: CHANGES.md [hunk 2]
```diff
@@ -6,10 +6,12 @@ Version 0.5.0 is the fifth release.
 
 This release includes some API changes:
 
+`cats.laws.discipline.eq` no longer provides `Eq` instances for `tuple2` and `tuple3`, these instances and together with some other new instances for `tuple`s are now provided by `cats.std.tuple` (through inheriting the instance trait defined in algebra 0.4.2).
```
Comment (travisbrown) [inline CHANGES.md]: Nitpicky but could you capitalize `Tuple2`, etc.? The formatting and context makes it look like you're referring to real types.

File: CHANGES.md [hunk 3]
```diff
@@ -6,10 +6,12 @@ Version 0.5.0 is the fifth release.
...
+
 * [#910](https://github.com/typelevel/cats/pull/910): Remove `Streaming` and `StreamingT`
 * [#967](https://github.com/typelevel/cats/pull/967): `product` and `map` can be implemented in terms of `ap`
 * [#970](https://github.com/typelevel/cats/pull/970): Renamed `Kleisli#apply`to `ap`
-
+* [#994](https://github.com/typelevel/cats/pull/994): updated to latest algebra (brought in all the new goodies)
```
Comment (travisbrown) [inline CHANGES.md]: Should you remove this line below now that you've included it here?

Comment (kailuowang) [inline CHANGES.md]: line below removed. thanks @travisbrown

**Comment by travisbrown** [comment]
👍, thanks @kailuowang!

**Comment by non** [comment]
👍 Thanks!

**Pull request merged**